lib/open_project/translations/helpers/tmp_directory.rb

```diff
@@ -0,0 +1,14 @@
+module TmpDirectory
+  def create_tmp_directory(delete: nil)
+    tmp_path = File.join(FileUtils.pwd, 'tmp')
+    if delete
+      FileUtils.rm_rf tmp_path
+    end
+    FileUtils.mkdir_p tmp_path
+    tmp_path
```

Comment:
I also thought about that but was afraid of the big block.. :)
And do you think that it would be a good idea to not delete the tmp directory when in debugging mode, i.e. passing a variable `debug` to that method?